src/content/posts/releases/mitmproxy52/index.md

```diff
@@ -10,55 +10,59 @@ author:
   twitter: martinplattnr
 ---
 
-We are excited to announce the release of [mitmproxy 5.2](https://github.com/mitmproxy/mitmproxy/releases/tag/v5.2).
-It comes with many improvements and bugfixes: 48 merged PRs with 211 commits from over 30 contributors - see the full changelog below.
+We are excited to announce the release of [mitmproxy 5.2](https://github.com/mitmproxy/mitmproxy/releases/tag/v5.2)! This release comes not only with a long list of improvements and bugfixes -- 48 PRs from over 30 contributors --, but also with the first fruits from this year's Google Summer of Code:
```

Comment:
In case you add the `--no-http2` flag below: I think the `,` in `48 PRs from over 30 contributors --,` is a typo?